packages/office-ui-fabric-react/src/components/ResizeGroup/ResizeGroup.Props.ts

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { ResizeGroup } from './ResizeGroup';
+
+export interface IResizeGroup {
+  measure?: () => void;
+}
+
+export interface IResizeGroupProps extends React.HTMLProps<ResizeGroup | HTMLElement> {
+
+  /**
+   * Optional callback to access the IResizeGroup interface. Use this instead of ref for accessing
+   * the public methods and properties of the component.
+   */
+  componentRef?: (component: IResizeGroup) => void;
+
+  /**
+   * Data to be passed to the onRenderData function
+  */
+  data?: any;
```

Comment:
We've discussed it quite a bit, and the switch to 2.3 seems to be the best time to do it. I'll open an issue for it once this has been merged in (assuming we're okay with this going in without type safety).